packages/bottom-tabs/src/types.tsx

```diff
@@ -106,7 +106,7 @@ export type BottomTabNavigationOptions = {
   /**
    * Text to show in a badge on the tab icon.
    */
-  tabBarBadge?: number | string;
+  tabBarBadge?: number | string | boolean;
```

Comment:
this should actually be:

`tabBarBadge?: number | string | true;`

Passing in a `false` bool will actually still render the dot